src/com/core/tictactoe/GamesController.java

```diff
@@ -1,6 +1,6 @@
 package com.core.tictactoe;
 
-public class GamesController {
+public class GamesController implements GamesTypes {
```

Comment:
This says GamesController is a type of GamesTypes?  
GamesTypes is a list of constants so could just defined as statics directly in GamesController.

http://www.javapractices.com/topic/TopicAction.do?Id=32
This says: `The static import feature should always be considered as a replacement for this practice. ('this practice' means the interface for constants you used)`

Reply:
Done - ready on master (removed interface, put constants as private static final strings in GamesController.)

Reply:
Relevant commit: https://github.com/Kotauror/TicTacToeJava/commit/f2f34502d3efd58e49eed9f5a0eb3f2b6c328502